test/api/controllers/serverController.test.js

```diff
@@ -108,11 +109,25 @@ describe('Test: server controller', () => {
     });
   });
 
+  describe('#throw', () => {
+    it('should throw an ExternalServiceError with right name, msg and code', () => {
+      try {
+        serverController.throw('elasticsearch_down', '{"status":"red"}');
```

Comment:
If `throw` does not throw an error, this test will silently pass but we actually want it to fail